manifest.json

```diff
@@ -3688,6 +3688,17 @@
                 }
             },
             "versions": {
+                "1.3.0": {
+                    "changelog": "Added config items for notification sounds",
+                    "releaseUrl": "https://github.com/badhaloninja/UserJoinLeaveNotifications/releases/tag/v1.3.0",
```

Comment:
Also the actual tag doesn't have a "v" prefix so the releaseUrl is 404

Reply:
Oops, fixed on the github side bc I want to keep my repos in the same format the link is valid now